final_prep/README.md

```diff
@@ -29,10 +29,10 @@ Take your time as you go back and refactor your exercises from each section. We'
 
 ## Gearing Up for Gear Ups
 
-Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
+Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry.
 
 - [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
+  - [ ] [⚡️Casey's Gear Up Assigment⚡️](https://gist.github.com/7d103ad1d5252288c27aac367d7faf6b.git)
```

Comment:
Excellent reflections, Casey. 100% with you on the societal scarcity mentality and how it manifests itself in our identities and actions. Your perspective is valued in these conversations at Turing.